addons/knowledge/models/knowledge_article.py

```diff
@@ -760,15 +750,24 @@ def move_to(self, parent_id=False, before_article_id=False, is_private=False):
                 )
             self_sudo = self.sudo()
             self_member = self_sudo.article_member_ids.filtered(lambda m: m.partner_id == self.env.user.partner_id)
-            member_command = [(2, member.id) for member in self_sudo.article_member_ids if member.partner_id != self.env.user.partner_id]
+
             if self_member:
-                member_command.append((1, self_member.id, {'permission': 'write'}))
+                write_member_command = [(1, self_member.id, {'permission': 'write'})]
```

Comment:
Shouldn't we use `Command.update` for better readability? (also for subsequent commands)

Reply:
I see that the knowledge code base uses mostly the "magic numbers" writing.
(`_inverse_is_user_favorite`, `_remove_member`, ...).

Maybe it's best to keep it like that for consistency.

Reply:
Maybe because "someone" did go through the python code and still does not use the commands.